modules/ztp-disconnected-environment-prereqs.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-disconnected-environment-prereqs_{context}"]
+= Prerequisites
```

Comment:
Can this module (title, too :smile:) stand alone? If not, I believe it needs to be part of a procedure.

Reply:
I understand your suggestion, but I don't think I can move this into a procedure. I've updated the title though. 

BTW if you want to see this content in it's original context it is here: https://docs.openshift.com/container-platform/4.9/installing/installing-mirroring-installation-images.html

This is a tricky topic. This whole procedure will be going away when the https://github.com/quay/openshift-mirror-registry is incorporated into OCP proper - v4.11 I believe.